### ISSUE TITLE
When using @ prefixed identifiers, the prefix should be removed in the generated param name

### ISSUE THREAD

**Issue by toptensoftware** [kasecato/vscode-docomment#86]
Eg: this parameter:

```
void DoSomething(bool @checked)
```

is incorrectly generated as:

```
/// <param name="@checked"></param>
```

instead of 

```
/// <param name="checked"></param>
```

**Comment by kasecato** [comment]
Thanks @toptensoftware , I just released [v0.1.10](https://marketplace.visualstudio.com/items?itemName=k--kato.docomment), please check it 👍

**Comment by toptensoftware** [comment]
Thanks... seems to be working fine now.
